### ISSUE TITLE
Timeout with Node Version <= 16

### ISSUE THREAD

**Issue by focafull** [typesense/typesense-js#118]
## Description

I set up my own self hosted typesense instance on a vps. Now when I use typesense-js with node version 16 or less I will always get a timeout. If I use Node 17 everything works fine. I guess it has to do something with my server setup or dns setup, since everything works fine when I start a local docker container. When I check the health endpoint I can curl it just fine. 
Does anyone else had this issue and knows what to do?

**Comment by Enragedsaturday** [comment]
I had all kinds of nonsense trying to set up a self hosted instance on a vps. Changing the port in the config file to port 80 and  the same in the client fixed the issues for me.

**Comment by lantien** [comment]
@Enragedsaturday you are right, the only solution i could find was to set the config on port 80, but in order to worker you need to NOT specify the port.

**Comment by tharropoulos** [comment]
Closing this issue since it hasn't seen any activity in a while. Please feel free to reopen if you have more information.
